.tav.yml

```diff
@@ -584,14 +584,18 @@ aws-sdk:
     include: '>=3.15.0 <4'
 
 '@aws-sdk/client-sqs':
-  versions: '3.15.0 || 3.58.0 || 3.171.0 || 3.257.0 || 3.335.0 || 3.429.0 || ^3.436.0'
+  # v3.446.0 and v3.447.0 (latest at 2023-11-09) switched the internal format from XML to JSON
+  # and localstack does not support it yet. We stop testing at v3.445.0 intil JSON is supported
+  # https://github.com/elastic/apm-agent-nodejs/issues/3716
+  versions: '3.15.0 || 3.67.0 || 3.179.0 || 3.261.0 || 3.342.0 || 3.436.0 || 3.445.0'
   commands:
     - node test/instrumentation/modules/@aws-sdk/client-sqs.test.js
   node: '>=14'
   # Test v3.15.0, current latest and 5 versions in between.
   update-versions:
     mode: max-5
     include: '>=3.15.0 <4'
+    exclude: '3.446.0 || 3.447.0'
```

Comment:
Fancy :)

Reply:
Yup, I had to take out the caret manually in the last version on the list so there's room for improvement